standards/using-3rd-party-open-source-products.md

```diff
@@ -0,0 +1,32 @@
+# Using 3rd-party open source products
+
+In order to make effective commercial decisions, it is important to recognise the characteristics of open source software that we find desirable. Many organisations ‘open’ their code, but monetise the software through means that restrict the freedom of the user. Some open source software does not have a strong community, and can be risky to use should problems arise which the MOJ does not understand how to resolve.
```

Comment:
I'm not entirely sure I agree with the implications of the phrase 'Some open source software does not have a strong community.' I'd say that while most open source does not have a strong community, conversely the long-tail model suggests that the majority of the most important core technologies have extensive communities, LAMP to name just four.